Gemfile.lock

```diff
@@ -182,4 +230,4 @@ DEPENDENCIES
   web-console (>= 3.3.0)
 
 BUNDLED WITH
-   1.14.6
```

Comment:
looks like we all need to `gem update bundler` 1.15.0 is the most current version